app/experimenter/experiments/tests/test_admin.py

```diff
@@ -258,11 +259,13 @@ def test_after_import_row(self):
         resource.after_import_row(row=test_row, row_result=None)
         post_branches = NimbusBranch.objects.filter(experiment=experiment)
         post_changes = NimbusChangeLog.objects.filter(experiment=experiment)
+        post_experiment = NimbusExperiment.objects.get(slug=experiment.slug)
 
+        self.assertEqual(post_experiment.reference_branch.slug, "control")
         self.assertEqual(len(branches), len(post_branches))
         self.assertEqual(len(post_branches), 2)
 
         self.assertGreaterEqual(len(post_changes), len(pre_changes))
         self.assertGreaterEqual(len(post_changes), num_changes)
 
-        self.assertIsNotNone(experiment.reference_branch)
+        self.assertIsNotNone(post_experiment.reference_branch)
```

Comment:
Isn't this check redundant with the one up on line 264?  By this point we already know it's not None because we've accessed `.slug` on it?

Reply:
Yep, good catch. This was an existing test that I updated, but should have realized it was made redundant by the previous one.